ios/safari_extension/Resources/manifest.json

```
@@ -0,0 +1,30 @@
+{
+	"manifest_version": 2,
+	"default_locale": "en",
+
+	"name": "@@PROJECTNAME@@",
+	"description": "An awesome safari extension!",
+	"version": "1.0",
+
+	"icons": {
+        "48": "images/icon-48.png",
+        "96": "images/icon-96.png",
+        "128": "images/icon-128.png",
+        "256": "images/icon-256.png",
+        "512": "images/icon-512.png"
+    },
+
+    "browser_action": {
+        "default_popup": "popup.html",
+        "default_icon": {
+            "16": "images/toolbar-icon-16.png",
+            "19": "images/toolbar-icon-19.png",
+            "32": "images/toolbar-icon-32.png",
+            "38": "images/toolbar-icon-38.png",
+            "48": "images/toolbar-icon-48.png",
+            "72": "images/toolbar-icon-72.png"
+        }
+    },
+
+    "permissions": [ ]
```

There’s a mix of tabs and spaces in this file that should be cleaned up

```suggestion
		"48": "images/icon-48.png",
		"96": "images/icon-96.png",
		"128": "images/icon-128.png",
		"256": "images/icon-256.png",
		"512": "images/icon-512.png"
	},

	"browser_action": {
		"default_popup": "popup.html",
		"default_icon": {
			"16": "images/toolbar-icon-16.png",
			"19": "images/toolbar-icon-19.png",
			"32": "images/toolbar-icon-32.png",
			"38": "images/toolbar-icon-38.png",
			"48": "images/toolbar-icon-48.png",
			"72": "images/toolbar-icon-72.png"
		}
	},

	"permissions": [ ]
```